README.md

```diff
@@ -10,81 +10,89 @@ in mind so it can be used as more than just GTD software.
 GTG is intended to help you track everything you need to do and need to know,
 from small tasks to large projects.
 
-## Dependencies
+## INSTALLING AND RUNNING
 
-GTG depends on the following packages:
+In order to download and run GTG Developer Version, do the following steps 
+(Debian-based systems):
 
- * Python, version 3.0 or above
- * PyGTK
- * python-support
- * python-xdg
- * python-dbus
- * python-liblarch 
- * yelp (to read GTG documentation)
+DOWNLOAD: 
+Execute this command to get the latest Developer package and then move to that directory:
 
-Please refer to your system documentation for information on how to install
-these modules if they're not currently available.
+	$ git clone https://github.com/getting-things-gnome/gtg.git
+
+  $ cd gtg
```

Comment:
Indent it properly please, so markdown parses it as code block (also drop the empty line above it)